### ISSUE TITLE
Setting `bundlePath` option dos not remove console warnings

### ISSUE THREAD

**Issue by nolder3193** [CloudCannon/pagefind#290]
I'm trying to hide the console warnings saying that the bundle path could not be resolved based on the javascript import path. The docs say that specifying the `bundlePath` option for PagefindUI should hide those warnings but it doesn't.

Code:
```javascript
import { PagefindUI } from '@pagefind/default-ui'; // resolves fine

// inside the `connect` method of a Stimulus controller
this.pagefindUI = new PagefindUI({
  baseUrl: '/',
  bundlePath: '/_pagefind/',
  element: `#${this.elementId}`,
  showImages: false,
  resetStyles: true
});
```

**Comment by bglw** [comment]
Sorry about this — this is a wee bug with the order of load/options for the underlying JS API. I have some plans to fix this for the upcoming Pagefind 1.0 so I'll make sure to test this specific case.

**Comment by bglw** [comment]
Hey @nolder3193 ! 👋

Good news — this has landed in Pagefind v1.0.0! ✨

See the full release notes here: https://github.com/CloudCannon/pagefind/releases/tag/v1.0.0 💙
